### PR TITLE
ECM-620

### PR DESCRIPTION
**Pull request by hsjhita**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ECM-620

### Change description ###

Added validation for multiples when transferring them to different offices

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleTransferService.java [hunk 1]
```diff
@@ -54,11 +56,13 @@ public void multipleTransferLogic(String userToken, MultipleDetails multipleDeta
...
         } else {
 
-            log.info("Send updates to single cases");
-
-            multipleDetails.getCaseData().setState(UPDATING_STATE);
+            validateCaseBeforeTransfer(userToken, multipleDetails, errors, multipleObjects);
 
-            sendUpdatesToSinglesCT(userToken, multipleDetails, errors, multipleObjects);
+            if (errors.isEmpty()) {
+                log.info("Send updates to single cases");
```
Comment (gauravchawla73463) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleTransferService.java:62]: May be remove this log message

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleTransferServiceTest.java [hunk 2]
```diff
@@ -134,4 +155,46 @@ public void populateDataIfComingFromCT() {
...
+
+        when(excelReadingService.readExcel(anyString(), anyString(), anyList(), any(), any()))
+                .thenReturn(multipleObjects);
+        when(singleCasesReadingService.retrieveSingleCases(anyString(), anyString(), anyList(), anyString()))
+                .thenReturn(new ArrayList<>(Collections.singletonList(submitEvent)));
+        doCallRealMethod().when(caseTransferService).validateCase(isA(CaseData.class), anyList());
+
+        caseTransferService.validateCase(caseData, errors);
+        assertEquals(2, errors.size());
+        assertEquals(String.format(BF_ACTIONS_ERROR_MSG, caseData.getEthosCaseReference()), errors.get(0));
```
Comment (gauravchawla73463) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleTransferServiceTest.java:195]: May be test the scenario when there are 0 errors

**Review by jthmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleTransferService.java (hunk 1 above)
Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleTransferService.java]: Removed

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleTransferServiceTest.java (hunk 2 above)
Comment (hsjhita) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleTransferServiceTest.java:201]: Scenario created to test 0 errors

**Review by gauravchawla73463 — APPROVED** [review]

**Pull request merged**